modules/ebi-metagenomics/jbrowse/sortgff/main.nf

```diff
@@ -0,0 +1,41 @@
+process TABIX_TABIX {
```

Comment:
This module is not running tabix, nf-core already has one https://nf-co.re/modules/tabix_tabix/